src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md

```diff
@@ -0,0 +1,341 @@
+
+    npx hardhat
+
+You should then see a welcome message and option to select what you want to do. Select “create an empty hardhat.config.js”:
+
+    888    888                      888 888               888
+    888    888                      888 888               888
+    888    888                      888 888               888
+    8888888888  8888b.  888d888 .d88888 88888b.   8888b.  888888
+    888    888     "88b 888P"  d88" 888 888 "88b     "88b 888
+    888    888 .d888888 888    888  888 888  888 .d888888 888
+    888    888 888  888 888    Y88b 888 888  888 888  888 Y88b.
+    888    888 "Y888888 888     "Y88888 888  888 "Y888888  "Y888
+    👷 Welcome to Hardhat v2.0.11 👷‍
+    ? What do you want to do? …
+    Create a sample project
+    ❯ Create an empty hardhat.config.js
+    Quit
+
+This will generate a hardhat.config.js file for us which is where we’ll specify all of the set up for our project (on step 13).
+
+## Step 9: Add project folders {#add-project-folders}
+
+To keep our project organized, we’ll create two new folders. Navigate to the root directory of your project in your command line and type:
+
+    mkdir contracts
+    mkdir scripts
+
+* contracts/ is where we’ll keep our NFT smart contract code
+
+* scripts/ is where we’ll keep scripts to deploy and interact with our smart contract
+
+## Step 10: Write our contract {#write-contract}
+
+Now time that our environment is set up, onto more exciting stuff: *writing our smart contract code!*
+
+Open up the my-nft project in your favorite editor (we like [VSCode](https://code.visualstudio.com/)). Smart contracts are written in a language called Solidity which is what we will use to write our MyNFT.sol smart contract.‌
+
+1. Navigate to the “contracts” folder and create a new file called MyNFT.sol
+
+2. Below is our NFT smart contract code, which based off of the [OpenZepplin](https://docs.openzeppelin.com/contracts/3.x/erc721) library’s ERC-721 implementation. Copy and paste the contents below into your MyNFT.sol file.
+
+    //Contract based on [https://docs.openzeppelin.com/contracts/3.x/erc721](https://docs.openzeppelin.com/contracts/3.x/erc721)
+    // SPDX-License-Identifier: MIT
+    pragma solidity ^0.7.3;
+
+    import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+    import "@openzeppelin/contracts/utils/Counters.sol";
+    import "@openzeppelin/contracts/access/Ownable.sol";
+    
+
+    contract MyNFT is ERC721, Ownable {
+        using Counters for Counters.Counter;
+        Counters.Counter private _tokenIds;
+
+        constructor() public ERC721("MyNFT", "NFT") {}
+
+        function mintNFT(address recipient, string memory tokenURI)
+            public onlyOwner
+            returns (uint256)
+        {
+            _tokenIds.increment();
+
+            uint256 newItemId = _tokenIds.current();
+            _mint(recipient, newItemId);
+            _setTokenURI(newItemId, tokenURI);
+
+            return newItemId;
+        }
+    }
+
```

Comment:
So our markdown parser can sometimes get the indentation syntax correct, but it's safer to wrap these code blocks in triple-backticks. You can also add the language that way to help with syntax coloring.

**This is how this section renders now:**
![image](https://user-images.githubusercontent.com/54227730/115784032-733e6400-a372-11eb-97d5-ca50a3071a0e.png)
**If you wrap this block (and others) in triple-backticks (on their own lines) and trail the opening set with the language name, like this...**
![image](https://user-images.githubusercontent.com/54227730/115784411-eba52500-a372-11eb-8087-0649eff01c2e.png)
**...then it will render like this:**
![image](https://user-images.githubusercontent.com/54227730/115784464-fe1f5e80-a372-11eb-8e1f-2ad92a8b898e.png)

Lets make sure all the code blocks are wrapped this way to minimize issues and keep it consistent =) thanks!